### DISCUSSION TITLE
Wallet Connect V1 Android SDK not working with latest version (6.5.0) of Metamask

### DISCUSSION THREAD

**Discussion by duybuikardiachain** [WalletConnect/walletconnect-monorepo#2372] [Developer Support]
WC1 not working with Metamask 6.5.0
But 6.3.0 is working as a normal
I already asking on Metamask gihub
Please help to check, still waiting Metamask support V2, but now I have to use V1
So boring...

**Comment by chendinh** [comment]
its not problem of walletconnect sir. Just use the right metamask's version
.

**Reply by duybuikardiachain** [reply]
I just updated latest version of metamask on CH Play. Hope anyone faced this issue, then know how to fix it.
Still waiting Metamask support WC2
